code/src/components/AddTaskForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+import { list, todolist } from 'reducers/todolist'
+import { taskContainer } from 'components/taskContainer.css'
+import { button } from 'components/button.css'
+
+
+
+export const AddTaskForm = () => {
+    const dispatch = useDispatch()
+
+    const [inputValue, setInputValue] = useState("")
+
+    const handleOnSubmit = (e) => {
+        console.log(inputValue)
```

Comment:
remove console log. 
Also, maybe add a check to see if the input is empty or not before adding it. Right now you can save an empty todo